index.html

```diff
@@ -3,31 +3,71 @@
-    <title>Quiz App</title>
+    <title>Quiz-App</title>
     <link rel="stylesheet" href="styles/index.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.1/css/all.css"
+      integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp"
+      crossorigin="anonymous"
+    />
   </head>
 
   <body class="app-grid">
     <header class="app-grid__header">
-      <h1>QUIZ-APP</h1>
+      <h1 class="page-title">QUIZ-APP</h1>
     </header>
 
     <main class="app-grid__main">
-      main
-
-      <div class="flaticon-referal">
-        Icons made by
-        <a href="https://www.freepik.com" title="Freepik">Freepik</a> from
-        <a href="https://www.flaticon.com/" title="Flaticon"
-          >www.flaticon.com</a
-        >
-      </div>
+      <section class="quiz-card">
+        <a class="fas fa-bookmark quiz-card__bookmark" href=""></a>
+        <h2 class="quiz-card__heading">QUESTION #1</h2>
+        <p class="quiz-card__question">What does SEO mean?</p>
+        <div class="quiz-card__solution">
+          <button class="quiz-card__solution__button">Show answer</button>
+          <p>Search Engine Optimization.</p>
+        </div>
+        <ul class="quiz-card__tag-area">
+          <li class="quiz-card__tag-area__tags">seo</li>
+          <li class="quiz-card__tag-area__tags">web development</li>
+          <li class="quiz-card__tag-area__tags">programming</li>
+          <li class="quiz-card__tag-area__tags">beginner</li>
+        </ul>
+      </section>
+      <section class="quiz-card">
+        <a class="fas fa-bookmark quiz-card__bookmark" href=""></a>
+        <h2 class="quiz-card__heading">QUESTION #2</h2>
+        <p class="quiz-card__question">What does SEO mean?</p>
+        <div class="quiz-card__solution">
+          <button class="quiz-card__solution__button">Show answer</button>
+          <p>Search Engine Optimization.</p>
+        </div>
+        <ul class="quiz-card__tag-area">
+          <li class="quiz-card__tag-area__tags">seo</li>
+          <li class="quiz-card__tag-area__tags">programming</li>
+          <li class="quiz-card__tag-area__tags">web development</li>
+        </ul>
+      </section>
+      <section class="quiz-card">
+        <a class="fas fa-bookmark quiz-card__bookmark" href=""></a>
+        <h2 class="quiz-card__heading">QUESTION #3</h2>
+        <p class="quiz-card__question">What does SEO mean?</p>
+        <div class="quiz-card__solution">
+          <button class="quiz-card__solution__button">Show answer</button>
+          <p>Search Engine Optimization.</p>
+        </div>
+        <ul class="quiz-card__tag-area">
+          <li class="quiz-card__tag-area__tags">web development</li>
+          <li class="quiz-card__tag-area__tags">seo</li>
+          <li class="quiz-card__tag-area__tags">beginner</li>
+        </ul>
+      </section>
     </main>
     <footer class="app-grid__footer navbar">
```

Comment:
Consider adding a `nav`-element around your anchors which you could give the class name `navbar`.
```suggestion
    <footer class="app-grid__footer">
      <nav class="navbar">
       ...
```